### DISCUSSION TITLE
Show names of disabled services ASUS (G814JV 2023)

### DISCUSSION THREAD

**Discussion by DripSweat** [seerge/g-helper#3311] [Q&A]
Good day.
I read the threads - did not see an answer.
Maybe the current thread should be moved to the "ideas" section...

The essence of the question:
The application g-helper allows you to stop the services that ASUS launched after installing the drivers. In my example, it looked like this:
8 ASUS services are running
Press the stop button in g-hepler
Then other information appears - the number of running services is zero.
However, if you open the task manager - you can see that one or more services remain enabled + applications ASUS.

![image](https://github.com/user-attachments/assets/5ff8c6e5-94d3-423c-b8f9-c2501cc019ce)
![image](https://github.com/user-attachments/assets/2d8854a3-9e18-40c3-9c02-d83fe9390aaa)

The question arises - what is stopped when you press the "STOP" key?
Maybe it is worth showing "somewhere at the bottom of the window" that this service is stopped, this one is running, etc.

**Comment by seerge** [comment]
@DripSweat those are not services, but some other asus apps. If you don't use them - you can uninstall them via add & remove programs

**Reply by DripSweat** [reply]
The question was different)
In the application, the G-helper shows 8 ASUS services
When the application stopped them, it shows that zero services are running
But in fact, one or more services are running.

The question arises - is it possible to display information about which specific services were stopped? (maybe in future releases)

**Reply by seerge** [reply]
App shows counter for [services that it can stop](https://github.com/seerge/g-helper/wiki/FAQ#what-services-can-be-stopped-from-the-extra-settings). When they are stopped it shows 0. Everything is correct here.

Apps on your screenshot are not services, and are not supposed to be stopped.

**Reply by DripSweat** [reply]
![image](https://github.com/user-attachments/assets/63e289c7-b44c-4db9-9f15-374c065a2ec1)

Judging by the clipping from the task manager, one of the services is running (ASUS HID Service)

**Reply by seerge** [reply]
Service with that specific name is not listed / stopped by the app (it's also not being part of Asus System Control Interface package that you usually have on your system, so I don't know what exactly it does, may be it's something important for Armoury Crate?). If you want you can stop it manually

**Reply by DripSweat** [reply]
Judging by what it does, it is responsible for the hot keys (M keys)

Then, if I understand correctly, your application stops all services related to Armory Crate, right?
Then there is no need to show anything... (since that's the situation)

**Reply by seerge** [reply]
Hotkeys are handled by AsusOptimization service. G-Helper does not stop AC services. It stops leftovers left after using official AC uninstaller - ArmouryCrateControl service (popup to install AC) + MyASUS services that can't be stopped anyhow else.

**Reply by seerge** [reply]
It could be responsible for that fancy numpad that Strix models have on touchpad tho

**Reply by DripSweat** [reply]
That's probably true)
I have exactly this model ROG Strix G814JV
